.circleci/config.yml

```diff
@@ -0,0 +1,76 @@
+version: 2.1
+jobs:
+  build:
+    working_directory: ~/comit
```

Comment:
❓ Where does this come from?

Reply:
What do you mean? This is just to set the path where to build things